packages/static-build/src/utils/gatsby.ts

```diff
@@ -353,3 +370,19 @@ export async function createPluginSymlinks(dir: string) {
     )
   );
 }
+
+function tryReadFileSync(filePath: string) {
+  try {
+    return fs.readFileSync(filePath, 'utf-8');
+  } catch (error: unknown) {
+    if (!isErrnoException(error)) {
+      console.error(error);
+    } else {
+      if (error.code !== 'ENOENT') {
+        console.error(error);
+      }
+    }
```

Comment:
```suggestion
    if (!isErrnoException(error) || error.code !== 'ENOENT') {
      console.error(error);
    }
```

Reply:
I think this is clearer: [`f6d2c12` (#9975)](https://github.com/vercel/vercel/pull/9975/commits/f6d2c121fb0e5b4d0645cf74e24078c30e674b49)